macaroonbakery/keys.py

```diff
@@ -5,24 +5,38 @@
 
 
 class PrivateKey(object):
+    ''' A 256-bit Ed25519 public key.
+    '''
     def __init__(self, key):
         self._key = key
 
     @property
     def key(self):
+        ''' Internal nacl key representation.
+        '''
         return self._key
 
     @property
     def public_key(self):
+        '''
+        :return: the PublicKey associated with the private key.
+        '''
         return PublicKey(self._key.public_key)
 
     @classmethod
     def deserialize(cls, serialized):
+        ''' Create a PrivateKey from a base64 encoded bytes.
```

Comment:
A public key used by the bakery to encrypt third party caveats.
Every discharger is associated with a public key which is
used to encrypt third party caveat ids addressed to that discharger.
Internally, it is a 256 bit Ed25519 public key.

?